src/rules/completedDocsRule.ts

```diff
@@ -109,7 +109,7 @@ export class Rule extends Lint.Rules.TypedRule {
         },
     };
 
-    public static ARGUMENT_DESCRIPTOR_BLOCK = {
+    public static ARGUMENT_DESCRIPTOR_BLOCK: {} = {
```

Comment:
this isn't even correct. it has a much more advanced type than the empty object.

Reply:
`{}` is a correct type. Anything but `null` or `undefined` or `void` is assignable to `{}`.